### DISCUSSION TITLE
Welcome to pymeasure Discussions!

### DISCUSSION THREAD

**Discussion by cjermain** [pymeasure/pymeasure#390] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bilderbuchi** [comment]
I am Christoph, a physicist by education and working in research outside academia, in Austria. I started using pymeasure to enable my colleagues to more easily interface with the many instruments in our lab. By now, I am also helping out maintaining pymeasure itself. Although life keeps me pretty busy elsewhere, i do what I can! :smiley:

**Comment by Bruyant** [comment]
Hi,

I'm a physicist by education now working on bulding experiment for solid state pysics in High magnetic field. I use this framework for some of my experiments.

So firts, i'd like to thanks all the contributor of this great library.

**Comment by BenediktBurger** [comment]
I am Benedikt, physicist as well, but still in academia (PhD student).
I switched the software of my project from Labview to Python and wrote all the drivers myself. Looking for the drivers of a newly arrived instrument, I found different driver libraries and wanted to participate in one project, to share my work. In the end I decided upon pymeasure, as it is active, the instrument design is simple (not the overhead of metaclasses etc. like in Lantz), such that it is easy to understand the instrument code (necessary for introducing new people, students etc.) and the design is flexible enough (instrumentkit uses the same command for read and write and assumes, that reading has always a question mark, further it does not distinguish between read and write termination, which is required for some instruments) to accomodate the devices I use.
A major minus of pymeasure has been the lack of easy device testing, which is fixed now with #634 😁

**Reply by bilderbuchi** [reply]
> A major minus of pymeasure has been the lack of easy device testing, which is fixed now with #634 😁

...which would not have happened by now without your initiative. Thank you! ❤️

**Comment by ricounet67** [comment]
I am Eric, electron microscopist and I want to use PyMeasure to control different sample holder.
I am a beginner in Python and willing to learn and progress
